airgun/entities/modulestream.py

```diff
@@ -52,9 +53,9 @@ def prerequisite(self, *args, **kwargs):
 
     def step(self, *args, **kwargs):
         entity_name = kwargs.get('entity_name')
-
+        stream_version = kwargs.get('stream_version')
         self.parent.search(
-            'name = {0}'.format(entity_name))
+            'name = {0} and stream = {1}'.format(entity_name, stream_version))
         self.parent.table.row(name=entity_name)['Name'].widget.click()
```

Comment:
perhaps we should also update this line to:
```python
self.parent.table.row(name=entity_name, stream=stream_vesrsion)['Name'].widget.click()
```

Reply:
Done!